# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/tsa/slrn/slrn-0.9.8.0-mime.patch
# ROCK Linux is Copyright (C) 1998 - 2004 Clifford Wolf
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- slrn/src/mime.c~	2002-11-01 13:32:34.000000000 +0100
+++ slrn/src/mime.c	2003-11-21 18:39:34.000000000 +0100
@@ -650,7 +650,7 @@
    l = a->lines;
    
    /* skip header and separator */
-   while (((l != NULL) && (l->flags & HEADER_LINE)) || l->buf[0] == '\0')
+   while ((l != NULL) && ((l->flags & HEADER_LINE) || l->buf[0] == '\0'))
      l = l->next;
    
    if (l == NULL) return;
